src/core/js/models/adaptModel.js

```diff
@@ -172,29 +183,27 @@ define([
         },
 
         findDescendants: function (descendants) {
-
+            var children = this.getChildren().models;
+            
             // first check if descendant is child and return child
             if (this._children === descendants) {
-                return this.getChildren();
+                return children;
             }
 
             var allDescendants = [];
             var flattenedDescendants;
-            var children = this.getChildren();
             var returnedDescedants;
```

Comment:
any chance you could correct the typo in this variable name while you're here? it's been annoying me for ages!